deployment/deployment.json

```diff
@@ -2925,7 +2925,7 @@
           },
           "decentralized-network": {
             "slug": "lido-ethereum",
-            "query-id": "F7qb71hWab6SuRL5sf6LQLTpNahmqMsBnnweYHzLGUyG"
+            "query-id": "todo"
```

Comment:
Why are you changing the query-id to `todo`? When re-deployed, it keeps the same query-id.